grc/utils/form_custom_validators.py

```diff
@@ -141,14 +135,6 @@ def password_check(password):
         raise ValidationError('Your password needs to contain 8 or more characters, a lower case letter, an upper case letter, a number and a special character')
 
 
-def validateAdopted(form, field):
-
-    # Custom validator for the "Are you adopted", StrictRequiredIf doesn't play nice with this question
-    if form['check'].data == 'Yes':
-        if field.data is None or field.data == '':
-            raise ValidationError('Select if your were you adopted in the United Kingdom')
-
-
```

Comment:
This code isn't used any more.
We seem to be using the `wtforms.validators.DataRequired` validator instead
see https://github.com/cabinetoffice/grc-app/blob/master/grc/birth_registration/forms.py#L106